server.js

```diff
@@ -54,6 +54,13 @@ router.get(['/', '/index.html'], (req, res) => {
   });
 });
 
+router.get(['/', '/volunteer.html'], (req, res) => {
```

Comment:
why the '/' here? shouldn't that be '/volunteer'? I think we can also get rid of the '/volunteer.html' bit and the array match. That was mostly for backwards compatibility for index.html and give.html in case they were linked/embedded from other sites

Reply:
fixed